lib/models/ecs-model-deployer.ts

```diff
@@ -69,10 +71,12 @@ export class ECSModelDeployer extends Construct {
             memorySize: 1024,
             role: role,
             environment: {
-                'LISA_VPC_ID': props.vpcId,
+                'LISA_VPC_ID': props.vpc.vpc.vpcId,
```

Comment:
Everywhere else we're doing `props.vpc?.blah`, should this not be `props.vpc?.vpc` as well if it is potentially undefined?